common/docker-compose.yaml

```diff
@@ -35,6 +35,9 @@ services:
       # The GCP KMS keyring name to use
       TF_VAR_keyring_name: keyring
 
+      TF_VAR_organization_id: ${ORGANIZATION_ID}
```

Comment:
I can go further with this and say – can we maybe use the same docker compose everywhere when `exekube` is needed? :)

Reply:
The original motivation for having separate (and different) `docker-compose.yaml` was because the `common/` side needs `~/.aws` mounted in while the `gcp/` side does not. (I think there were 1-2 other differences as well, but I don't think I see them now. Perhaps the compose.env abstraction allows these differences to be hidden?)

If that mount is the only remaining difference between common/d-c.yaml and gcp/d-c.yaml, perhaps it is worth unifying the two files. `~/.aws` can be left (or forced to be) empty on the gcp/ side.

Alfredo, are we missing anything from the last time we discussed this?